### ISSUE TITLE
q-field: Unexpected Data Clearing with clearable Attribute Enabled

### ISSUE THREAD

**Issue by FangHanZreo** [quasarframework/quasar#17113]
### What happened?

Clicking on a QField component will clear the model value.### 

### What did you expect to happen?

Only upon clicking the clear button should the model value be cleared.

### Reproduction URL

https://codepen.io/fanghanzreo/pen/MWRqRYg

### How to reproduce?

Click q-filed component 

### Flavour

Quasar CLI with Webpack (@quasar/cli | @quasar/app-webpack)

### Areas

Components (quasar)

### Platforms/Browsers

Microsoft Edge

### Quasar info output

```Shell
» Dev mode.................. spa
 » Pkg quasar................ v2.15.0
 » Pkg @quasar/app-webpack... v3.12.4
 » Transpiled JS............. yes (Babel)
```

**Comment by myleslee** [comment]
It happens on Quasar 2.15.3 as well.

**Comment by myleslee** [comment]
@rstoenescu this issue prevents us from upgrading. Will it be merged soon? Thanks

**Comment by rstoenescu** [comment]
Pushed fix. Will be available in Quasar v2.16.2.
